src/navigations/HomeBottomTabs.js

```diff
@@ -127,15 +138,19 @@ function HomeBottomTabs() {
           component={ChannelListScreen}
           initialParams={{isBottomTab: true}}
           listeners={({route}) => saveLastMenu(route)}
-          options={{tabBarBadge: signedChannelUnreadCount > 0 ? signedChannelUnreadCount : null}}
+          options={{
+            tabBarBadge: signedChannelUnreadCount > 0 ? signedChannelUnreadCount : null,
+            tabBarBadgeStyle: styles.bottomBarBadge
+          }}
         />
         <Tab.Screen
           name="AnonymousChannelList"
           component={AnonymousChannelListScreen}
           initialParams={{isBottomTab: true}}
           listeners={({route}) => saveLastMenu(route)}
           options={{
-            tabBarBadge: anonymousChannelUnreadCount > 0 ? anonymousChannelUnreadCount : null
+            tabBarBadge: anonymousChannelUnreadCount > 0 ? anonymousChannelUnreadCount : null,
+            tabBarBadgeStyle: styles.bottomBarBadge
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `tabBarBadgeStyle` option has been added to the options for the 'SignedChannelList' and 'AnonymousChannelList' screens. This allows for custom styling of the badge that indicates unread messages.
```diff
+ tabBarBadgeStyle: styles.bottomBarBadge
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->